connectors/sources/atlassian.py

```diff
@@ -60,3 +62,114 @@ async def _remote_validation(self, advanced_rules):
         return SyncRuleValidationResult.valid_result(
             SyncRuleValidationResult.ADVANCED_RULES
         )
+
+
+class AtlassianAccessControl:
+    def __init__(self, source, client):
+        self.source = source
+        self.client = client
+
+    def prefix_account_id(self, account_id):
+        return prefix_identity("account_id", account_id)
+
+    def prefix_group_id(self, group_id):
+        return prefix_identity("group_id", group_id)
+
+    def prefix_role_key(self, role_key):
+        return prefix_identity("role_key", role_key)
+
+    def prefix_account_name(self, account_name):
+        return prefix_identity("name", account_name.replace(" ", "-"))
+
+    def access_control_query(self, access_control):
+        return es_access_control_query(access_control)
+
+    async def fetch_all_users(self, url):
+        start_at = 0
+        while True:
+            async for users in self.client.api_call(url=f"{url}?startAt={start_at}"):
+                response = await users.json()
+                yield response
+                if len(response) == 0:
+                    return
```

Comment:
```suggestion
                if len(response) == 0:
                    return
                yield response
```

Question: Should it be this way? So we're not yielding a `response`, when it doesn't contain anything.

Reply:
Updated! Not yielding empty response